package in.co.rays.junit;

import org.junit.Test;

public class TsetFail {

	public void testAdd() throws Exception {
@Test
		UserBean bean = new UserBean();
		bean.setId(11);
		bean.setName("abc");
		bean.setSalary(6000);
		UserModel model=new UserModel();
		model.add(bean);
		bean=UserModel.findByPk(12);
		if(bean==null) {
			System.out.println("Record is not added");
		}else
		{
			System.out.println("Record is added");
		}
	}

}
